docs/rules/no-trailing-spaces.md

```diff
@@ -1,12 +1,14 @@
-# Disallow trailing spaces at the end of lines (no-trailing-spaces)
+# disallow trailing whitespace at the end of lines (no-trailing-spaces)
 
 (fixable) The `--fix` option on the [command line](../user-guide/command-line-interface#fix) automatically fixes problems reported by this rule.
 
-Sometimes in the course of editing files, you can end up with extra whitespace at the end of lines. These whitespace differences can be picked up by source control systems and flagged as diffs, causing frustration for developers. While this extra whitespace causes no functional issues, many code conventions require that trailing spaces be removed before checkin.
+Sometimes in the course of editing files, you can end up with extra whitespace at the end of lines. These whitespace differences can be picked up by source control systems and flagged as diffs, causing frustration for developers. While this extra whitespace causes no functional issues, many code conventions require that trailing spaces be removed before check-in.
 
 ## Rule Details
 
-The following patterns are considered problems:
+This rule disallow trailing whitespace at the end of lines.
```

Comment:
What do you think about deleting the comment in incorrect code below and adding it here?

This rule disallows trailing whitespace (spaces, tabs, and other Unicode whitespace characters) at the end of lines.